api/python/cellxgene_census/tests/test_open.py

```diff
@@ -442,7 +441,7 @@ def test_opening_census_without_anon_access_fails_with_bogus_creds() -> None:
     os.environ["AWS_SECRET_ACCESS_KEY"] = "fake_key"
     # Passing an empty context
     with pytest.raises(
-        (tiledb.TileDBError, soma.DoesNotExistError),
+        soma.DoesNotExistError,
```

Comment:
Just an FYI: while this test passes fine when using the latest versions of tiledb-py and tiledbsoma-py (0.33.6 and 1.16.1) installed from PyPI (as confirmed in the CI in this repo), I observed that this test still raised an error of type `tiledb.TileDBError` when using the latest versions of tiledb-py and tiledbsoma-py installed as conda-forge binaries.

I don't know if this is a cause for concern. If not, please ignore my comment. More details in https://github.com/conda-forge/cellxgene-census-feedstock/pull/3